src/js/components/LoginPage/Login.scss

```diff
@@ -0,0 +1,127 @@
+$primary-color: #fff;
+
+.login{
+  background-color: #bbb;
+  min-height: 100vh;
+  color: $primary-color;
+  padding-bottom: 30px;
+  & span {
+    display: block;
+    overflow: hidden;
+    padding: 0 4px 0 6px
+  }
+}
+
+label {
+  float: left
+}
+
+input {
+  width: 100%
+}
+.form-input{
+  background-color: transparent;
+  border: 1px solid $primary-color;
+  font-size: 15px;
+  height: 35px;
+  padding-left: 10px;
+  color: $primary-color;
+}
+
+.margin-top-10{
+  margin-top: 10px;
+}.margin-top-20{
+  margin-top: 20px;
+}
+.margin-left-20{
+  margin-left: 20px;
+}
+
+.margin-top-30{
+  margin-top: 30px;
+}
+
+.btn-login-options{
+  border-radius: 100%;
+  border: 1px solid grey;
+  margin-right: 8px;
+  width: 30px;
+  height: 30px;
+  &.facebook{
```

Comment:
These are all getting the same attribute and value assigned to them combine them into a single declaration

Reply:
Fixed